features/old/proxy.feature

```diff
@@ -66,16 +66,15 @@ Feature: Proxy integration
     Given I'm using a custom API Backend
     Then I should be able to switch back to using the default API Backend
 
-
-  @javascript
+  @javascript @selenium
```

Comment:
No need of `@selenium` anymore, any `@javascript` is run under a real browser

Reply:
Sounds good!